### PR TITLE
problem fixed

### PR DESCRIPTION
**Pull request by YogevSwisa**
closing #67

### CODE REVIEW

**Comment by dumblob** [comment]
Sorry it took me this long to answer. Actually this sounds similar to https://github.com/dumblob/mysql2sqlite/issues/66 - would the solution in https://github.com/dumblob/mysql2sqlite/issues/66#issuecomment-573446979 help? Could you please test it?

**Comment by YogevSwisa** [comment]
hey, the solution in #66 not solves the problem.

**Comment by dumblob** [comment]
Ok, I looked at the mysql doc and you're totally right. I think your patch looks good for this purpose (it shouldn't break other functionality). Could you provide a test case and write it to `unit_tests.sh`? Then I'll merge this :wink:. Thanks.

**Comment by YogevSwisa** [comment]
i added a test case.😉

File: unit_tests.sh [hunk 1]
```diff
@@ -128,3 +128,13 @@ CREATE TABLE `scimag` (
...
+
+cat <<\SQL
+CREATE TABLE `scimag` (
+  `ID` int(15) unsigned NOT NULL AUTO_INCREMENT,
+  `DOI` varchar(200) NOT NULL,
+  `TEXTFIELD` TEXT DEFAULT ('text_value'),
+  PRIMARY KEY (`ID`) USING BTREE,
+  UNIQUE KEY `DOIUNIQUE` (`DOI`) USING BTREE,
+);
+SQL
```
Comment (dumblob) [inline unit_tests.sh]: Hm, I can't see any `_utf8mb3` inside this section - at best please provide directly the SQL source you've used to discover this issue.

Please also rename `SQL` heredoc to `SQLin` and then add another section `SQLout` with the correct output from this script :wink:.

Then I'll merge this good find of yours. Thanks in advance!

Comment (YogevSwisa) [reply unit_tests.sh]: Oops 😅, i forgot to put the sql after the mysqldump. 
SQLin - fixed.

Thanks.

**Comment by dumblob** [comment]
Thanks!

**Pull request merged**